libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -164,6 +164,7 @@
     private boolean browserLoginEnabled;
     private boolean idpLoginFlowEnabled;
     private String idpAppURIScheme;
+    private boolean idpAppLoginFlowActive;
```

Comment:
Flag to indicate if IDP login flow is active that helps components decide whether to switch user or not.